### ISSUE TITLE
WebOS 4.5 Software Version 5.30.40 43UM7050PLF

### ISSUE THREAD

**Issue by Gulli03** [RootMyTV/RootMyTV.github.io#122]
Is this version patched? It seems to be the case.
If it is, is there any way to downgrade the version? I have read somewhere that downgrading is possible for this software version if you have a eu device. Any help is appreciated.

**Comment by throwaway96** [comment]
Yes, it's patched. Downgrading from that version is not possible (at least without already having root, AccessUSB, or direct eMMC/bootloader access), EU device or not. There is at least one public privilege escalation vulnerability though.

**Comment by Gulli03** [comment]
> There is at least one public privilege escalation vulnerability though.
what exactly does this mean? Can I root my TV with this?

**Comment by throwaway96** [comment]
See the [crashd guide](https://gist.github.com/throwaway96/e811b0f7cc2a705a5a476a8dfa45e09f).
